### PR TITLE
Remove unused import

### PR DESCRIPTION
**Pull request by TrapinchO**
Removed unused imports from these files:

> core/events/custom/GameEvent.java
> 
> core/events/custom/WynnSocialEvent.java
> 
> 
> modules/core/instances/packet/PacketIncomingFilter.java
> modules/core/overlays/ui/ModConflictScreen.java
> 
> modules/map/rendering/PointRenderer.java
> 
> modules/utilities/instances/InfoFormatter.java
> modules/utilities/managers/NametagManager.java
> 
> modules/utilities/overlays/inventories/SkillPointOverlay.java
> 
> modules/visual/overlays/ui/CharacterSelectorUI.java
> 
> 
> webapi/profiles/guild/GuildProfile.java
> webapi/profiles/FileUpdater.java
> webapi/profiles/LeaderboardProfile.java
> webapi/profiles/ServerProfile.java
> webapi/profiles/UpdateProfile.java

Also added "A Marker Interface" to the "Listener" interface

### CODE REVIEW

**Review by magicus — APPROVED** [review]
Looks good. Github says there is a merge conflict; resolve it and I'll push the PR.

**Comment by TrapinchO** [comment]
> Looks good. Github says there is a merge conflict; resolve it and I'll push the PR.

The problem is I have no idea why. Maybe I am just idiot, but I really have no idea.

**Comment by TrapinchO** [comment]
I could make redo it of course

**Comment by TrapinchO** [comment]
Stackoverflow is the saviour of all programmers. Like always

File: src/main/java/com/wynntils/core/framework/interfaces/Listener.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /*
  *  * Copyright © Wynntils - 2018 - 2021.
+ *
+ * Marker interface
```
Comment (HeyZeer0) [inline src/main/java/com/wynntils/core/framework/interfaces/Listener.java]: Wrong position, this is inside the copyright notice

**Review by HeyZeer0 — CHANGES_REQUESTED** [review]
LGTM after the change is done

**Review by magicus — APPROVED** [review]

**Pull request merged**